$NetBSD: patch-source_API_CMakeLists.txt,v 1.3 2019/10/19 14:01:37 adam Exp $

Fix finding libxml2's includes.
Disable exports file for SunOS builds.

--- source/API/CMakeLists.txt.orig	2019-07-10 11:09:29.000000000 +0000
+++ source/API/CMakeLists.txt
@@ -13,6 +13,10 @@ if(LLDB_BUILD_FRAMEWORK)
   set(option_install_prefix INSTALL_PREFIX ${LLDB_FRAMEWORK_INSTALL_DIR})
 endif()
 
+if(LIBXML2_FOUND)
+  include_directories(SYSTEM ${LIBXML2_INCLUDE_DIR})
+endif()
+
 add_lldb_library(liblldb SHARED
   SBAddress.cpp
   SBAttachInfo.cpp
@@ -133,7 +137,7 @@ set_target_properties(liblldb
   VERSION ${LLDB_VERSION}
 )
 
-if (NOT CMAKE_SYSTEM_NAME MATCHES "Windows")
+if (NOT ( CMAKE_SYSTEM_NAME MATCHES "Windows" OR CMAKE_SYSTEM_NAME MATCHES "SunOS"))
   if (NOT LLDB_EXPORT_ALL_SYMBOLS)
     # If we're not exporting all symbols, we'll want to explicitly set
     # the exported symbols here.  This prevents 'log enable --stack ...'
